osu.Game.Rulesets.Catch/Difficulty/CatchDifficultyCalculator.cs

```diff
@@ -41,6 +41,8 @@ protected override DifficultyAttributes Calculate(IBeatmap beatmap, Mod[] mods,
             var catcher = new CatcherArea.Catcher(beatmap.BeatmapInfo.BaseDifficulty);
             float halfCatchWidth = catcher.CatchWidth * 0.5f;
 
+            halfCatchWidth *= 0.8f;
```

Comment:
Can be directly calculated during initialisation by replacing `0.5f` with `0.4f`.

Reply:
I believe doing it like this makes the intention more clear: showing we only use 80% of the catcher width. Making halfCatchWidth 40% of the full width would look like a typo/bug.

Reply:
it's more clear if a comment is added directly above this line explaining why it exists ;).